### PR TITLE
Reviews sub-collection for games

### PR DESCRIPTION
**Pull request by cr4zy30**

### CODE REVIEW

**Review by YousefAbdelnour — APPROVED** [review]
awesome

**Review by YousefAbdelnour — CHANGES_REQUESTED** [review]
same thing. follow my countries games PR for the same format

File: app/src/Models/GameModel.php [hunk 1]
```diff
@@ -28,4 +30,14 @@ public function getGameById($game_id): mixed
...
         return $this->fetchSingle($sql, ["game_id" => $game_id]);
     }
+
+    public function getReviewsByGameId($game_id): array
+    {
+        $query_args = [];
+
+        //TODO: same fileting logic for get /reviews
+
+        $sql = "SELECT * FROM review WHERE game_id = :game_id";
```
Comment (YousefAbdelnour) [inline app/src/Models/GameModel.php:38]: Get genre dev and country info along with review and game

**Review by YousefAbdelnour — CHANGES_REQUESTED** [review]
do those changes ples

**Comment by cr4zy30** [comment]
Here's what the response looks like after including the developer, country, and genre
image

**Review by YousefAbdelnour — APPROVED** [review]
LGTM👌

**Comment by YousefAbdelnour** [comment]
We have merge conflict. don't know how to solve it. Up to you @cr4zy30

**Pull request merged**